modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/variables/_navs.scss

```diff
@@ -0,0 +1,387 @@
+// nav-base-link
+
+$nav-base-link: (
+	after: (
+		bottom: 0,
+		content: none,
+		display: block,
+		position: absolute,
+		left: 0,
+		right: 0,
+		width: auto,
+	),
+);
+
+// Primary tab
+
+$nav-underline-link: (
+	active-class: (
+		background-color: var(--tab-primary-item-background-color-active),
+		border-color: var(--tab-primary-item-border-color-active),
+		box-shadow: var(--tab-primary-item-box-shadow-active),
+		color: var(--tab-primary-item-color-active),
+		font-weight: var(--tab-primary-item-font-weight-active),
+		after: (
+			background-color: var(--tab-primary-item-active-bar-color-active),
+			height: var(--tab-primary-item-active-bar-thickness),
+			content: '',
+		),
+		inline-item: (
+			color: var(--tab-primary-item-icon-color-active),
+		),
```

Comment:
As a team we need to discuss how we want to sort objects. Can you raise this in our next meeting?